### PR TITLE
fixes an exploit with cancer

### PR DESCRIPTION
**Pull request by Intigracy**

### CODE REVIEW

**Comment by ghost** [comment]
Fair enough

**Comment by Megamaw** [comment]
what the fuck kind of exploit is there with cancer

**Comment by kol1th** [comment]
@Megamaw Read the commit jackass

**Comment by Megamaw** [comment]
@kol1th Do I look like a guy that knows things

**Comment by kol1th** [comment]
@Megamaw You do not

**Comment by clusterfack** [comment]
:ok_hand:

**Comment by jknpj** [comment]
So, now you need +FUN to give people cancer and not only +ADMIN?

**Comment by Intigracy** [comment]
Yeah with the admin checks, you need one of the two listed, not both.

Which is how admins without FUN give people cancer.

**Comment by ComicIronic** [comment]
NO
ONE
LIFE
IFS

**Comment by Intigracy** [comment]
blame dylan

File: code/datums/datumvars.dm [hunk 1]
```diff
@@ -543,7 +554,8 @@ body
 		href_list["datumrefresh"] = href_list["give_spell"]
 
 	else if(href_list["addcancer"])
-		if(!check_rights(R_ADMIN|R_FUN))	return
+		if(!check_rights(R_FUN))
+			return
```
Comment (Intigracy) [inline code/datums/datumvars.dm:558]: only line actually changed

**Pull request merged**